### DISCUSSION TITLE
budibase-client.js load timing out

### DISCUSSION THREAD

**Discussion by podtris** [Budibase/budibase#8552] [Help]
Hello all

I've installed Budibase on a server and the backend is working fine. However, when I try to publish or preview the app, it comes up with:

There was an error loading your app
The Budibase client library could not be loaded. Try republishing your app.

Looking at it it is because the budibase-client.js file is too big, so the browser doesn't load the full file. Then as it isn't a full javascript file, it comes up with "Uncaught SyntaxError: Unexpected end of input (at budibase-client.js?v=2.1.15:13:545100)"

Has anyone got any suggestions please? Budibase looks to be an ideal solution for me but if we can't get this sorted I'll have to go down another route.

Many thanks
Podtris

**Comment by melohagan** [comment]
Hey @podtris 

Sorry you're having issues. Would you be able to email your app export to support@budibase.com ?
Then I can take a look.

**Reply by podtris** [reply]
All sorted @melohagan, I had a few problems with the app but once I'd upped the server memory from 2GB to 4GB everything seems to be working OK 😃

**Reply by melohagan** [reply]
Fantastic news! :)

**Reply by podtris** [reply]
sorry @melohagan, I spoke too soon, it's completely broken now and won't let me login from any browser, it seems to be truncating the /builder/assets/index.76f0215e.js file, causing it to error. Will raise a support ticket email.

**Reply by shogunpurple** [reply]
@podtris

Can you please provide more information on the following:
- How did you hosting budibase (docker-compose, single image, CLI etc)
- The server/OS/Version you are using

**Reply by podtris** [reply]
thanks @shogunpurple, I'm using the pre-packaged DigitalOcean image which says it is Budibase 1.0.0 on Ubuntu 20.04

**Reply by podtris** [reply]
@shogunpurple I've sorted this by building an image from scratch (looks like the image on DigitalOcean is out of date)? However, I still have the problem listed in my original question post above, with the full javascript file not loading?

This is the bottom of the file here:
![image](https://user-images.githubusercontent.com/75067443/201126072-d2c20996-4207-4cfc-9302-908d5fc86fd4.png)

No amount of changing browsers, PCs, removing caches etc seems to sort it

Any ideas please?

Many thanks
